pandas/tests/series/test_operators.py

```diff
@@ -1857,3 +1857,15 @@ def test_op_duplicate_index(self):
         result = s1 + s2
         expected = pd.Series([11, 12, np.nan], index=[1, 1, 2])
         assert_series_equal(result, expected)
+
+    def test_argminmax(self):
+        # GH13595
+
+        # Expected behavior for arg min/max in the presence of NA and Inf
+        s = pd.Series([0, -np.inf, np.inf, np.nan])
+
+        assert s.argmin() == 1
+        assert np.isnan(s.argmin(skipna=False))
+
```

Comment:
Ah, I didn't even know about this! So based on the notes on the [options documentation page](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.set_option.html), it looks like Pandas used to treat floating point +/- inf, as well as nan, as missing values by default, and this behavior was changed at some point. That's a big help; will check this and update.